bower.json

```diff
@@ -16,4 +16,4 @@
   "resolutions": {
     "route-recognizer": "^0.2.7"
   }
-}
+}
```

Comment:
Yes, it is. VS Code doesn't take the .editorconfig by default, but I've got the extension installed now.